manifests/profile/rabbitmq.pp

```diff
@@ -10,10 +10,35 @@
     }
   }
 
-  class { '::nova::rabbitmq':
-    userid             => $::openstack::config::rabbitmq_user,
-    password           => $::openstack::config::rabbitmq_password,
-    cluster_disk_nodes => [$management_address],
-    rabbitmq_class     => '::rabbitmq',
+  # Ideally we would configure the firewall to open the rabbitmq ports
+  # and allow access to the rabbitmq nodes, but we can't restrict the
+  # clustered node ports with early version of the rabbitmq module and
+  # therefore we can't open those ports. Instead we just ensure that
+  # requests on localhost use the localhost network. We will replace
+  # this with firewall rules when all the stackforge modules have
+  # upgraded to puppetlabs-rabbitmq >=5.x
+  host { 'localhost':
```

Comment:
This entry may be managed elsewhere outside the module by a customer. Can't we assume that `localhost` will just work?